dp/cloud/python/magma/configuration_controller/tests/unit/test_dp_logs.py

```diff
@@ -114,3 +119,6 @@ def test_dp_log_created_from_db_response(self, with_cbsd, serial_num, fcc_id, ne
     def test_make_dp_log_returns_type_error_for_unknown_message_type(self):
         with self.assertRaises(TypeError):
             make_dp_log(IncorrectDPLog())
+
+    def test_datetime_now(self):
+        self.assertEqual(SOME_DATE, now())
```

Comment:
Isn't this only testing that the `@freeze_time` decorator works? I think that is already ensured because otherwise the `self.assertEqual(expected_log, actual_log)` lines should fail, right?